node/pegnet/grading.go

```diff
@@ -80,8 +86,13 @@ func (p *Pegnet) InsertRates(tx *sql.Tx, height uint32, rates []opr.AssetUint, p
 			ratePEG.SetUint64(rates[i].Value)
 			continue
 		}
-		// Correct rates to use `pAsset`
-		rates[i].Name = "p" + rates[i].Name
+		if rates[i].Name == "pUSD" {
```

Comment:
do you think it's worth the effort to migrate the db to remove this oddity? (ie strip the 'p' prefix from existing rates)

Reply:
The confusing thing is that pUSD has a price, and USD has a price. If we are storing rates, they should be stored as the asset that was actually pulled.

Right now it's a bit ambiguous if we store pAsset rates with Asset prices. But then again the logic is that the Asset price is the pAsset price for conversions. So I just added a `exch_pAsset` to indicate it's the pAsset price on an exchange. 

A migration has a bit more work involved, and I can grasp the current understanding and make sense of it. So if we were to restart, I'd like to strip the 'p'. But at the current state, I can live with it.

Reply:
I agree, it makes sense if USD is the price of USD determined by miners, and pUSD is the market price of pUSD.

It's probably not worth the effort do it at this point but if we ever move to a database v5, we should do it.